code-reviews/process-guidance/reviewer-guidance.md

```diff
@@ -23,11 +23,14 @@ Code reviews should use the below guidance and checklists to ensure positive and
 
 - Be positive – encouraging, appreciation for good practices.
 - Avoid language that points fingers like “you” but rather use “we” or “this line” -- code reviews are not personal and language matters.
+- Prefer asking questions above making statements. There might be a good reason for the author to do something.
+- If you make a direct comment, explain why the code needs to be changed, preferably with an example.
+- If a few back-and-forth comments don't resolve a disagreement, have a quick talk with eachother (in-person or call). Don't forget to update the PR with what you agreed on and why.
 
 ### Make comments clear
 
 - Explain why the code needs to change.
-- Prefix a “point of polish” with “Nit:”.
+- Prefix a “point of polish” with “Nit:” - or use the Azure DevOps `suggestion` button and propose the correction directly.
 - If one or two comments don’t resolve a disagreement, talk in person or on the phone.
```

Comment:
Github also has this option  - may be it could be rephrased more broadly?

Reply:
actually i did as we recommend here:
I did 2 things: 1. committed suggestion directly to your branch and 2. created PR to your branch.  
I think 2(PR way) is preferable. 
I however do not see "accept changes" directly in PR that i have seen before.

Reply:
Good point, thank you! I think that your changes are now reflected in this PR, so all good :).

Reply:
I missed this conversation - @zoekdestep please ignore my overlapping/conflicting suggestion